nodebuilder/fraud/module.go

```diff
@@ -16,12 +16,12 @@ func Module(tp node.Type) fx.Option {
 	case node.Light:
 		return fx.Module(
 			"fraud",
-			fxutil.ProvideAs(ServiceWithSyncer, new(fraud.Service), new(fraud.Subscriber)),
+			fxutil.ProvideAs(ServiceWithSyncer, new(fraud.Service)),
```

Comment:
you can just do fxutil.Provide(ServiceWithSyncer) bc it already returns a fraud.Service

same with the case for full and bridge

Reply:
fx.Provide *

Reply:
Yes, thank you.